cmd/imagec/docker.go

```diff
@@ -144,16 +145,17 @@ func FetchToken(url *url.URL) (*Token, error) {
 }
 
 // FetchImageBlob fetches the image blob
-func FetchImageBlob(options ImageCOptions, image *ImageWithMeta) error {
+func FetchImageBlob(options ImageCOptions, image *ImageWithMeta) (*string, error) {
```

Comment:
I prefer "" vs a pointer to a string.  I think we do that in other places as well.